docker-for-windows/release-notes.md

```diff
@@ -434,6 +434,21 @@ We did not distribute a 1.12.4 stable release
 
 ## Edge Release Notes
 
+### Docker Community Edition 17.11.0-ce-rc3-win38 2017-11-09 (Edge)
+
+* Upgrades
+  - [Docker 17.11.0-ce-rc3](https://github.com/docker/docker-ce/releases/tag/v17.11.0-ce-rc3)
+
+* New
+  - Use Microsoft localhost port forwarder for Windows and LCOW Containers when it is available (insider build RS4)
```

Comment:
I'm not familiar with the term, "insider build". Just making sure this is what you meant and not "inside".